proto/message.py

```diff
@@ -388,6 +392,11 @@ def to_json(
             use_integers_for_enums (Optional(bool)): An option that determines whether enum
                 values should be represented by strings (False) or integers (True).
                 Default is True.
+            including_default_value_fields (Optional(bool)): Deprecated. Use argument
+                `always_print_fields_with_no_presence` instead. An option that
+                determines whether the default field values should be included in the results.
+                Default is True. If `always_print_fields_with_no_presence` is set to False, this
```

Comment:
Both of these fields say that they don't take effect if the other is False. This is confusing enough that it's not obvious what the right semantics are and whether they make sense.

I recommend setting the default value for both fields in the arg list above to be `None`, and then using them thus:

```
if ((always_print_fields_with_no_presence is not None) and 
    (including_default_value_fields is not None) and
    (always_print_fields_with_no_presence != including_default_value_fields):
   raise ValueError("new and deprecated parameters are inconsistent")

print_field = (always_print_fields_with_no_presence if always_print_fields_with_no_presence is not None
               else (including_default_value_fields if including_default_value_fields is not None
                     else True)

```

and simplifying the comments for both parameters to say that any value explicitly provided for one must be consistent with the any value explicitly provided for the other.